CHANGELOG.md

```diff
@@ -42,6 +42,8 @@
   [#11244](https://github.com/Kong/kong/pull/11244)
 - Add beta support for WebAssembly/proxy-wasm
   [#11218](https://github.com/Kong/kong/pull/11218)
+- Fixed critical level logs when starting external plugin servers. Those logs cannot be suppressed due to the limitation of OpenResty. We choose to remove the socket availibilty detection feature.
```

Comment:
This entry should move to the new `unrelease` section after 3.4 officially release.

Reply:
Not resolved?